Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2010,25 +2010,18 @@ extension BrowserViewController {
         // False = .variant 2 which is our new Intro View that we are A/B testing against
         // and get that from the server
         guard LeanPlumClient.shared.getSettings() != nil else {
-            Sentry.shared.send(message: "Onboarding Research: LP State - \(LeanPlumClient.shared.lpState.rawValue)", tag: .leanplum, severity: .debug, description: "Condition: Leanplum is disabled")
+            Sentry.shared.send(message: "Onboarding Research: onStartLPVariable - LP State - \(LeanPlumClient.shared.lpState.rawValue) | Condition: Leanplum is disabled", tag: .leanplum, severity: .debug, description: "Condition: Leanplum is disabled")
             self.onboardingUserResearch?.updateValue(value: true)
             showProperIntroVC()
             return
         }
-        // forceVariableUpdate - Thie method from Leanplum is used
-        // to re-sync variable and ensures that if any variable
-        // has changed on the Leanplum server side we receive
-        // variable changed callback
-        // Ref: https://docs.leanplum.com/reference#callbacks
-        Sentry.shared.send(message: "Onboarding Research: LP State - \(LeanPlumClient.shared.lpState.rawValue)", tag: .leanplum, severity: .debug, description: "Forcing variable update from LP")
-        LeanPlumClient.shared.forceVariableUpdate()
         // Condition: Update from leanplum server
         // Get the A/B test variant from leanplum server
         // and update onboarding user reasearch
-        onboardingUserResearch?.updatedLPVariables = {(lpVariable) -> () in
+        LeanPlumClient.shared.onStartLPVariable = {(lpVariable) -> () in
```

Comment:
Trying out getting a callback after leanplum has finished starting. If there is a race condition this should take care of it and in sentry we should see logs for it.